src/vendor/handler.js

```diff
@@ -20,8 +20,8 @@ function acknowledgeDelivery(orderId) {
 }
 
 
-module.exports = { acknowledgeDelivery,
+module.exports = { toTest: { acknowledgeDelivery,
   sendPickup,
-  events, };
+  events, } };
```

Comment:
Get rid of the `toTest` part because when it exporting those functions and events, your index.js was not reading it or running it